<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1161</width>
    <height>678</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>710</width>
    <height>437</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="opaqueResize">
       <bool>true</bool>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="widgetLeftMenu" native="true">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="widgetLOGO" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <spacer name="horizontalSpacer_8">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>45</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btnTitleIcon">
             <property name="minimumSize">
              <size>
               <width>155</width>
               <height>86</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>1111111</width>
               <height>11111</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_9">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>45</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTreeWidget" name="treeWidget">
          <column>
           <property name="text">
            <string>1</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="CustomTitleWidget" name="widgetTital" native="true"/>
        </item>
        <item>
         <widget class="QStackedWidget" name="stackedWidget">
          <property name="currentIndex">
           <number>7</number>
          </property>
          <widget class="ResourceManageDialog" name="m_ResourceManageDialog"/>
          <widget class="InformationConfihurationDialog" name="m_InforConfihurationDialog"/>
          <widget class="FtpDialog" name="m_FtpDialog"/>
          <widget class="ApprovalProgressDialog" name="m_ApprovalProgressDialog">
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
           </layout>
          </widget>
          <widget class="QWidget" name="page4">
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="MyTabWidget" name="tabWidgetModulel1">
              <property name="tabShape">
               <enum>QTabWidget::Rounded</enum>
              </property>
              <property name="currentIndex">
               <number>0</number>
              </property>
              <property name="iconSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="elideMode">
               <enum>Qt::ElideMiddle</enum>
              </property>
              <widget class="QWidget" name="tab">
               <attribute name="title">
                <string>+</string>
               </attribute>
               <layout class="QGridLayout" name="gridLayout_6">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <property name="horizontalSpacing">
                 <number>0</number>
                </property>
                <item row="0" column="0">
                 <widget class="QWidget" name="widgetSize1" native="true">
                  <layout class="QGridLayout" name="gridLayout_5"/>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="tab_2">
               <attribute name="title">
                <string>Tab 2</string>
               </attribute>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page5">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="MyTabWidget" name="tabWidgetModulel2">
              <property name="currentIndex">
               <number>0</number>
              </property>
              <property name="elideMode">
               <enum>Qt::ElideMiddle</enum>
              </property>
              <widget class="QWidget" name="tab_3">
               <attribute name="title">
                <string>+</string>
               </attribute>
               <layout class="QGridLayout" name="gridLayout_4">
                <item row="0" column="0">
                 <widget class="QWidget" name="widgetSize2" native="true">
                  <layout class="QGridLayout" name="gridLayout_7"/>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="tab_4">
               <attribute name="title">
                <string>Tab 2</string>
               </attribute>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page6">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="MyTabWidget" name="tabWidgetModulel3">
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="tab_5">
               <attribute name="title">
                <string>+</string>
               </attribute>
              </widget>
              <widget class="QWidget" name="tab_6">
               <attribute name="title">
                <string>Tab 2</string>
               </attribute>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page7">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="MyTabWidget" name="tabWidgetModulel4">
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="tab_7">
               <attribute name="title">
                <string>+</string>
               </attribute>
               <layout class="QGridLayout" name="gridLayout">
                <item row="0" column="0">
                 <widget class="QWidget" name="widgetSize4" native="true">
                  <layout class="QGridLayout" name="gridLayout_9"/>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="tab_8">
               <attribute name="title">
                <string>Tab 2</string>
               </attribute>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1161</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionM1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>功能样机</string>
   </property>
  </action>
  <action name="actionM2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>几何样机</string>
   </property>
  </action>
  <action name="actionM3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>性能样机</string>
   </property>
  </action>
  <action name="actionM4">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>生产样机</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ResourceManageDialog</class>
   <extends>QWidget</extends>
   <header>ResourceManageDialog.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>InformationConfihurationDialog</class>
   <extends>QWidget</extends>
   <header>InformationConfihurationDialog.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ApprovalProgressDialog</class>
   <extends>QWidget</extends>
   <header>ApprovalProgressDialog.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FtpDialog</class>
   <extends>QWidget</extends>
   <header>FtpDialog.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CustomTitleWidget</class>
   <extends>QWidget</extends>
   <header>CustomTitleWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MyTabWidget</class>
   <extends>QTabWidget</extends>
   <header>mainwindow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
